### ISSUE TITLE
assembly descriptors should be optional

### ISSUE THREAD

**Issue by tdiesler** [rhuss/docker-maven-plugin#66]
The target image may just want to set a few env entries or a different command
 
```
[ERROR] Failed to execute goal org.jolokia:docker-maven-plugin:0.10.4:build (build-docker-image) on project wildfly-camel-itests-openshift-domain: Error reading assembly: No assembly descriptors found. -> [Help 1]
```

**Comment by jgangemi** [comment]
it looks like i forget to do this w/ #64 but i just submitted a pull request for this.

**Comment by rhuss** [comment]
fixed in 0.11.2
